test-utils/mocks/mockMatchMedia.ts

```diff
@@ -0,0 +1,33 @@
+import {act} from '@testing-library/react';
+
+export function mockMatchMedia(matches = false) {
+    const ogMatchMedia = window.matchMedia;
```

Comment:
What does `og` mean?

Reply:
og = original

https://www.merriam-webster.com/dictionary/OG

but i guess it was bad decision for var